src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
```

Comment:
- [ ] In order to remove all the CRA boilerplate that is no longer used, kindly delete the following files:

  - App.test.js
  - logo.svg
  - reportWebVitals.js
  - setupTests.js
  - public/logo192.png
  - public/logo512.png
  - public/manifest.json
  - public/robots.txt

That way, we will have a cleaner repo without unused files! 💪